tests/integration/test_pagure.py

```diff
@@ -173,6 +173,9 @@ def test_get_web_url(self):
         url = self.ogr_project.get_web_url()
         assert url == "https://pagure.io/ogr-tests"
 
+    def test_full_repo_name(self):
+        assert self.ogr_project.full_repo_name == "ogr-tests"
```

Comment:
You can try also some Pagure project with namespace.

You can find something here: https://pagure.io/browse/projects/

Reply:
Also, try some `fork` as well, please. The fork handling is done in a different way than in GitLab/GitHub.